src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyles = {};
+
+  sourceString.split(';').forEach((item) => {
+    const pairs = item.trim().split(':');
+
+    if (pairs.length === 2) {
+      const key = pairs[0].trim();
+      const value = pairs[1].trim();
+
+      if (key !== undefined && value !== undefined) {
```

Comment:
```suggestion
      if (key && value) {
```
You don't need to compare with `undefined` https://developer.mozilla.org/en-US/docs/Glossary/Truthy